refactor useChatbot to optimize state updates based on messageHistory

nazmul-rion opened the pull request.

nazmul-rion committed:
refactor useChatbot to optimize state updates based on messageHistory

nazmul-rion committed:
bump version to 2.2.7 in package.json

moinsam commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gigalogy/react-chatbot-kit-extension",
-  "version": "2.2.6",
+  "version": "2.2.7",
```
You should include the updated `.lock` file as well

nazmul-rion committed:
update yarn.lock file and add dependency on saveMessages call

nazmul-rion committed:
refactor: remove build file

nazmul-rion committed:
refactor: remove build file from the git

moinsam commented on package.json:
```diff
@@ -1,6 +1,6 @@
+  "version": "2.2.7",
   "publishConfig": {
     "access": "public"
   },
```
"homepage": "https://gigalogy.github.io/react-chatbot-kit-extension-docs/",

nazmul-rion committed:
chore: update homepage URL in package.json

moinsam committed:
Update workflows for manual trigger and yarn usage

Enable workflow_dispatch for manual runs in the build workflow. Switch from npm to yarn in the npm-publish workflow and add build step before publishing.

alifhsgigalogy committed:
Merge pull request #18 from gigalogy/feature/issue-86-moin

Update workflows for manual trigger and yarn usage
